file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [16-Apr-2025 15:29:47 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dagon-design-sitemap-generator-plus/sitemap-generator.php on line 722
[16-Apr-2025 15:29:54 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dagon-design-sitemap-generator-plus/sitemap-generator.php on line 722
[16-Apr-2025 15:30:02 UTC] PHP Fatal error:  Array and string offset access syntax with curly braces is no longer supported in /wordpress/wp-content/plugins/dagon-design-sitemap-generator-plus/sitemap-generator.php on line 722
[16-Apr-2025 15:30:49 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE `wp_api_status` (   `id`  int(11)   NOT NULL auto_increment,   `date`  date   NOT NULL,   `status`  tinyint(4)  NOT NULL,   PRIMARY KEY `order_id` (`id`) ) ENGINE=MyISAM DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE `w...')
#2 /wordpress/wp-content/plugins/dynamic-post/wp_plugin_dynamic_post.php(60): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): WP_Plugin_Dynamic_Post::activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_dynami...', false)
#7 /internal/eval.php(20): activate_plugin('dynamic-post/wp...')
#8 {main}</pre>
 for query CREATE TABLE `wp_api_status` (   `id`  int(11)   NOT NULL auto_increment,   `date`  date   NOT NULL,   `status`  tinyint(4)  NOT NULL,   PRIMARY KEY `order_id` (`id`) ) ENGINE=MyISAM DEFAULT CHARSET=latin1 AUTO_INCREMENT=1  made by activate_plugin, do_action('activate_dynamic-post/wp_plugin_dynamic_post.php'), WP_Hook->do_action, WP_Hook->apply_filters, WP_Plugin_Dynamic_Post::activate, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:30:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_api_status WHERE `date`= :param0  AND `status`= 1 | parameters: 2025-04-16</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')
#2 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(32): wpdb->get_results('SELECT * FROM w...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:30:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:30:51', '1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_api_status` (`date`, `status`) VALUES (:param0 , :param1 ) | parameters: 2025-04-16 15:30:51, 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_api_status', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(37): wpdb->insert('wp_api_status', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(727): do_action('init')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:30:51', '1') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:30:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_api_status WHERE `date`= :param0  AND `status`= 1 | parameters: 2025-04-16</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')
#2 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(32): wpdb->get_results('SELECT * FROM w...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:30:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:30:51', '1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_api_status` (`date`, `status`) VALUES (:param0 , :param1 ) | parameters: 2025-04-16 15:30:51, 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_api_status', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(37): wpdb->insert('wp_api_status', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(727): do_action('init')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:30:51', '1') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:00 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE TABLE `wp_api_status` (   `id`  int(11)   NOT NULL auto_increment,   `date`  date   NOT NULL,   `status`  tinyint(4)  NOT NULL,   PRIMARY KEY `order_id` (`id`) ) ENGINE=MyISAM DEFAULT CHARSET=latin1 AUTO_INCREMENT=1 </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Cannot combine AUTOINCREMENT and multiple primary keys in SQLite.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-admin/includes/upgrade.php(3280): WP_SQLite_DB->query('CREATE TABLE `w...')
#2 /wordpress/wp-content/plugins/dynamic-post/wp_plugin_dynamic_post.php(60): dbDelta(Array)
#3 /wordpress/wp-includes/class-wp-hook.php(324): WP_Plugin_Dynamic_Post::activate(false)
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_dynami...', false)
#7 /internal/eval.php(20): activate_plugin('dynamic-post/wp...')
#8 {main}</pre>
 for query CREATE TABLE `wp_api_status` (   `id`  int(11)   NOT NULL auto_increment,   `date`  date   NOT NULL,   `status`  tinyint(4)  NOT NULL,   PRIMARY KEY `order_id` (`id`) ) ENGINE=MyISAM DEFAULT CHARSET=latin1 AUTO_INCREMENT=1  made by activate_plugin, do_action('activate_dynamic-post/wp_plugin_dynamic_post.php'), WP_Hook->do_action, WP_Hook->apply_filters, WP_Plugin_Dynamic_Post::activate, dbDelta, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_api_status WHERE `date`= :param0  AND `status`= 1 | parameters: 2025-04-16</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')
#2 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(32): wpdb->get_results('SELECT * FROM w...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:31:01', '1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_api_status` (`date`, `status`) VALUES (:param0 , :param1 ) | parameters: 2025-04-16 15:31:01, 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_api_status', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(37): wpdb->insert('wp_api_status', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(727): do_action('init')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:31:01', '1') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT * FROM wp_api_status WHERE `date`= :param0  AND `status`= 1 | parameters: 2025-04-16</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT * FROM w...')
#2 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(32): wpdb->get_results('SELECT * FROM w...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query SELECT * FROM wp_api_status WHERE `date`= '2025-04-16' AND `status`= 1 made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:31:01', '1')</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: INSERT INTO `wp_api_status` (`date`, `status`) VALUES (:param0 , :param1 ) | parameters: 2025-04-16 15:31:01, 1</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_api_status.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(2614): WP_SQLite_DB->query('INSERT INTO `wp...')
#2 /wordpress/wp-includes/class-wpdb.php(2503): wpdb->_insert_replace_helper('wp_api_status', Array, NULL, 'INSERT')
#3 /wordpress/wp-content/plugins/dynamic-post/post-types/post_type_dynamic_post.php(37): wpdb->insert('wp_api_status', Array)
#4 /wordpress/wp-includes/class-wp-hook.php(324): Post_Type_Dynamic_Post->init('')
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-settings.php(727): do_action('init')
#8 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#9 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#10 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#11 {main}</pre>
 for query INSERT INTO `wp_api_status` (`date`, `status`) VALUES ('2025-04-16 15:31:01', '1') made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, Post_Type_Dynamic_Post->init, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[16-Apr-2025 15:31:10 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php:655
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('dashboard-widge...')
#2 /internal/eval.php(20): activate_plugin('dashboard-widge...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php on line 655
[16-Apr-2025 15:31:18 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php:655
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('dashboard-widge...')
#2 /internal/eval.php(20): activate_plugin('dashboard-widge...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php on line 655
[16-Apr-2025 15:31:26 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php:655
Stack trace:
#0 /wordpress/wp-admin/includes/plugin.php(2387): include_once()
#1 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('dashboard-widge...')
#2 /internal/eval.php(20): activate_plugin('dashboard-widge...')
#3 {main}
  thrown in /wordpress/wp-content/plugins/dashboard-widget-manager/dashboard-widget-manager.php on line 655
[16-Apr-2025 15:32:16 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>meta-box</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[16-Apr-2025 15:32:16 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_form.php on line 225
[16-Apr-2025 15:32:16 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_phone.php on line 37
[16-Apr-2025 15:32:25 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>meta-box</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[16-Apr-2025 15:32:25 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_form.php on line 225
[16-Apr-2025 15:32:25 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_phone.php on line 37
[16-Apr-2025 15:32:34 UTC] PHP Notice:  Function _load_textdomain_just_in_time was called <strong>incorrectly</strong>. Translation loading for the <code>meta-box</code> domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the <code>init</code> action or later. Please see <a href="https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 6.7.0.) in /wordpress/wp-includes/functions.php on line 6121
[16-Apr-2025 15:32:34 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_form.php on line 225
[16-Apr-2025 15:32:34 UTC] PHP Deprecated:  Required parameter $shortcode_name follows optional parameter $content in /wordpress/wp-content/plugins/debtcom-business-in-a-box/inc/bib_phone.php on line 37
[16-Apr-2025 15:34:53 UTC] PHP Fatal error:  Uncaught Error: Non-static method DineApp::get_default_widget_config() cannot be called statically in /wordpress/wp-content/plugins/dineapp/class-dineapp.php:501
Stack trace:
#0 /wordpress/wp-content/plugins/dineapp/class-dineapp.php(532): DineApp::get_widget_config()
#1 /wordpress/wp-content/plugins/dineapp/index.php(22): DineApp::register_sidebar_widget()
#2 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')
#3 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('dineapp/index.p...')
#4 /internal/eval.php(20): activate_plugin('dineapp/index.p...')
#5 {main}
  thrown in /wordpress/wp-content/plugins/dineapp/class-dineapp.php on line 501

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #5 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"dineapp"},"options":{"activate":true}}) : PHP.run() failed with exit code 255 and the following output: 
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      ... 2 lines matching cause stack trace ...
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: PHPExecutionFailureError: PHP.run() failed with exit code 255 and the following output: 
        at PHP.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/universal/index.js:987:19)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async wr (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:33:39)
        at async Object.gt [as installPlugin] (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:974:42)
        at async g (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12517:30)
        at async Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12446:23)
        at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
        at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
        at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
        at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
      response: PHPResponse {
        httpStatusCode: 500,
        headers: {
          'x-powered-by': [ 'PHP/8.0.30-dev' ],
          'content-type': [ 'text/html; charset=UTF-8' ]
        },
        bytes: Uint8Array(0) [],
        exitCode: 255,
        errors: ''
      },
      source: 'request'
    }
  }
}

Node.js v23.11.0
